driver/src/models/order.rs

```diff
@@ -41,4 +42,17 @@ pub mod test_util {
             sell_amount: 4,
         }
     }
+
+    pub fn order_to_executed_order(
```

Comment:
Should this be using the From or Into trait instead?

Reply:
Doesn't work because there are additional parameters.

Reply:
Even if it did work I am not sure if its correct to go into the whole From machinery when its not used this way.